brainstormforce/ultimate-addons-for-gutenberg
# Image block

### Description

New Block for an image. This Block is only for showcasing image with some necessary info(title, caption)



### Types of changes
New Link Components Added
New Image Block Added

### How has this been tested?
This is just another simple block. Used to showcase just image. Here is some test case below
1. Multi Type Image Upload (direct upload, default uploader, upload from URL)
2. Check all toolbar settings like alignment, duotone filter, cropping, etc
3. Check all general & styles settings 
4. Default and Custom mask settings with mask position

### Checklist:
